README.md

```diff
@@ -78,6 +78,8 @@ And one of those (on Linux):
 - having an `/etc/*-release` file that is compliant to the [OS-Release Spec](https://www.freedesktop.org/software/systemd/man/os-release.html) (and does not include `lsb`)
 - manually specify which version & system should be used
 
+On Linux, you will also need `libcurl4`. This will probably only be an issue on "slim" Docker images.
```

Comment:
```suggestion
On Linux, you will also need `libcurl4` (or `libcurl3` on some older distro versions). This will probably only be an issue on "slim" Docker images.
```